app/validators/schema_validator.rb

```diff
@@ -28,10 +28,13 @@ def valid?
 
   def schema
     @schema || find_schema
+  rescue NoSchemaNameError
+    errors << no_schema_name_message
+    {}
   rescue Errno::ENOENT => error
     if Rails.env.development?
       errors << missing_schema_message
-      errors << dev_help
+      errors << dev_help if GovukSchemas::Schema.all.empty?
```

Comment:
I think this will be redundant, as at least in the "development" RAILS_ENV, the Publishing API service will fail to start if this is the case https://github.com/alphagov/publishing-api/blob/master/config/environments/development.rb#L46